### PR TITLE
CONN-1220: Updated Direct backing beans to have single calls to Direct C...

### PR DESCRIPTION
**Pull request by sailajaa**
...onfig services

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectCertBean.java [hunk 1]
```diff
@@ -92,6 +84,7 @@ public void addCertificate() {
         certificate.setStatus(EntityStatus.NEW);
         addCert.getCerts().add(certificate);
         directService.addCertificate(addCert);
+        refreshCertificates();
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectCertBean.java:87]: Should this be here or where the action is called?  why are we refreshing when you upload a file to the UI?

Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectCertBean.java:87]: Ignore comment.  I missed which method this was called from.  diff threw me off.

Comment (sailajaa) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectCertBean.java:87]: My Understanding is, I am refreshing to get list of certificates after add certificates action.

Comment (vsaibabu) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectCertBean.java:87]: I am good with this PR, Merging it now.

**Pull request merged**